focuspoints.lrdevplugin/ShowMetadata.lua

```diff
@@ -84,24 +84,30 @@ function splitForColumns(metaData)
     if (v == nill) then v = "" end
     l = LrStringUtils.trimWhitespace(l)
     v = LrStringUtils.trimWhitespace(v)
-    
+
     labels = labels .. l .. "\r"
     values = values .. v .. "\r"
   end
   return labels, values
-  
+
 end
 
 function createParts(metaData)
   local parts = {}
   local num = 0;
-  for i in string.gmatch(metaData, "[^\\\n]+") do 
+  for i in string.gmatch(metaData, "[^\\\n]+") do
     log("i = " .. i)
-    p = splitText(i, ":")
-    if (p ~= nill) then
+
+    p = splitToKeyValue(i, ":")
+    if p ~= nill then
       parts[num] = p
-      num = num+1
+      num = num + 1
```

Comment:
I prefer the metadata display as currently in master. Would like to see this backed out.